### DISCUSSION TITLE
How to Use this ?

### DISCUSSION THREAD

**Discussion by waseemsammaai** [Tyrrrz/YoutubeDownloader#421] [Q&A]

**Comment by Tyrrrz** [comment]
You just run the `YoutubeDownloader.exe`

**Reply by woshixiaocaaa** [reply]
There is no such exe file

**Reply by Tyrrrz** [reply]
https://github.com/Tyrrrz/YoutubeDownloader/releases/latest

**Reply by woshixiaocaaa** [reply]
> https://github.com/Tyrrrz/YoutubeDownloader/releases/latest

The win64 version has an exe file, but arm64 does not have this file

**Reply by Tyrrrz** [reply]
It should have three, actually. If you don't see it, make sure your antivirus didn't quarantine it by accident.

**Reply by woshixiaocaaa** [reply]
> 实际上应该有三个。如果你没有看到它，请确保你的防病毒软件没有意外隔离它。

okay, thank you. It can sometimes be used, but errors often occur and it cannot be used. Why is this
